### PR TITLE
Feat/updated services acc. to new schema

### PR DESCRIPTION
**Pull request by SusantoMandal**

### CODE REVIEW

File: src/evaluation/evaluate-score.service.ts [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
 import { Question } from 'src/test/dto/question.dto';
 import { EvaluationProducerService } from './producer/evaluation-producer.service';
 import {
   EvaluatedAnswer,
-  Evaluation,
+  EvaluatedTestStatus,
 } from './interfaces/evaluated-test.interface';
 
-enum Status {
+enum TestStatus {
```
Comment (ashmeetkaur-compro) [inline src/evaluation/evaluate-score.service.ts:15]: Can create enum file seperated and keep all these enums in that file like enums -> test.enum.ts

File: src/evaluation/evaluate-score.service.ts [hunk 2]
```diff
@@ -31,39 +38,50 @@ export class EvaluateScoreService {
...
-        const evaluatedAnswer: EvaluatedAnswer = {
-          questionId: answer?.questionId,
-          questionScore: this.getScore(currQuestion, answer),
-          selectedAnswer: this.getSelectedOption(currQuestion, answer),
-        };
-        rawScore += evaluatedAnswer.questionScore;
-        evaluatedAnswers.push(evaluatedAnswer);
-      });
+          if (!currQuestion) {
+            throw new NotFoundException('Question not found in DB');
```
Comment (ashmeetkaur-compro) [inline src/evaluation/evaluate-score.service.ts:54]: Isnt it 'Question not found in text in DB'

File: src/evaluation/evaluate-score.service.ts (hunk 1 above)
Comment (SusantoMandal) [reply src/evaluation/evaluate-score.service.ts]: created `eums` folder and moved the code inside the suggested file `test.enum.ts`

File: src/evaluation/evaluate-score.service.ts (hunk 2 above)
Comment (SusantoMandal) [reply src/evaluation/evaluate-score.service.ts:46]: No. It is same as previous message `Question not found in DB`
image

**Pull request merged**